week5_R_draft.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
<p>If you can add a comment about how cbind binds the dataframe and how it does it so it is crystal clear the way the function works</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>That's a good point, I will, thanks!</p>